internal/cmptest/common_options.go

```diff
@@ -61,11 +66,31 @@ func CommonOptions(licenseCmp LicenseComparer, locationCmp LocationComparer) []c
 				return true
 			},
 		),
+		cmp.Comparer(
+			func(x, y pkg.CopyrightsSet) bool {
```

Comment:
This should be split out to a separate function, following the same pattern as the other comparers